scripts/portconfig

```diff
@@ -22,15 +22,20 @@ import argparse
 import swsssdk
 
 PORT_TABLE_NAME = "PORT"
+PORTCHANNEL_TABLE_NAME = "PORTCHANNEL"
+PORT_PREFIX_ETHERNET = "Ethernet"
```

Comment:
Please use the API's defined in https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-py-common/sonic_py_common/interface.py. This prevents duplicate definitions of table names and prefixes.

Reply:
Done.